src/System.IO.FileSystem.DriveInfo/tests/System.IO.FileSystem.DriveInfo.Tests.csproj

```diff
@@ -21,7 +20,7 @@
       https://github.com/dotnet/corefx/issues/564
       https://github.com/dotnet/corefx/issues/592
     -->
-    <RunTestsForProject>False</RunTestsForProject>
+    <RunTestsForProject>True</RunTestsForProject>
```

Comment:
It is not required although your OuterLoop test might never be run or might always be run if you don't specify it.